examples/blog-with-comment/components/comment/list.js

```diff
@@ -0,0 +1,49 @@
+import distanceToNow from '../../lib/dateRelative'
+import { useAuth0 } from '@auth0/auth0-react'
+
+function CommentList({ comments, onDelete }) {
+  const { user } = useAuth0()
+
+  return (
+    <div className="space-y-6 mt-10">
+      {comments.map((comment) => {
+        const isAuthor = user && user.sub === comment.user.sub
+        const isAdmin =
+          user && user.email === process.env.NEXT_PUBLIC_AUTH0_ADMIN_EMAIL
+
+        return (
+          <div key={comment.created_at} className="flex space-x-4">
+            <div className="flex-shrink-0">
+              <img
```

Comment:
Can this use `next/image`?

Reply:
Different auth providers have different image origins. That’s why I could not use it.